src/MaterialTabBar/Indicator.tsx

```diff
@@ -19,19 +19,21 @@ const Indicator: React.FC<IndicatorProps> = ({
   const opacity = useSharedValue(fadeIn ? 0 : 1)
 
   const stylez = useAnimatedStyle(() => {
-    const transform =
-      itemsLayout.length > 1
-        ? [
-            {
-              translateX: interpolate(
+    const transform = [
+      {
+        translateX:
+          itemsLayout.length > 1
+            ? interpolate(
                 indexDecimal.value,
                 itemsLayout.map((_, i) => i),
                 // when in RTL mode, the X value should be inverted
                 itemsLayout.map((v) => (isRTL ? -1 * v.x : v.x))
-              ),
-            },
-          ]
-        : undefined
+              )
+            : isRTL
```

Comment:
```suggestion
            : isRTL && itemsLayout[0]?.x
```

Because on the line below you are doing -1 * itemsLayout[0]?.x, which can result nan so check also if it exists

Reply:
This suggestion doesn't seem correct. What if `x` is 0?

May need to check for `typeof itemsLayout[0]?.x === 'number'` instead